kore/test/Test/Kore/Step/Simplification/IntegrationProperty.hs

```diff
@@ -55,7 +55,7 @@ import Test.SMT
 
 test_simplifiesToSimplified :: TestTree
 test_simplifiesToSimplified =
-    testPropertyWithSolver "simplify returns simplified pattern" $ do
+    testPropertyWithSolver "zzzsimplify returns simplified pattern" $ do
```

Comment:
Does the `zzz` mean anything?

Reply:
Sorry, I use that to make it easier to run only one test.